__tests__/connectRoutes.js

```diff
@@ -114,7 +114,13 @@ describe('middleware', () => {
     expect(location.kind).toEqual('redirect')
     expect(location.type).toEqual('THIRD')
     expect(history.entries.length).toEqual(2)
-    expect(location).toMatchSnapshot()
+    expect(location.prev).toEqual({
```

Comment:
ok remove all test related code back to your original PR. Then make sure on the "conversation" tab I'm added in the top right with the ability to edit your PR.